cairo_programs/cairo_0/ecdsa_builtin.cairo

```diff
@@ -0,0 +1,12 @@
+%builtins ecdsa
+
+from starkware.cairo.common.cairo_builtins import SignatureBuiltin
+
+func main{ecdsa_ptr: SignatureBuiltin*}() {
+    %{ ecdsa_builtin.add_signature(ids.ecdsa_ptr.address_, (ids.signature_r, ids.signature_s)) %}
```

Comment:
where does this `signature` come from? (I know we talked on the tact that we currently don't run hint, but here even with hints it wouldn't run)

Reply:
Indeed, I was inlining the [`verify_ecdsa_signature` function](https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/cairo/common/signature.cairo#L6-L19) and forgot about the signature variables